contracts/lease/src/position/dto.rs

```diff
@@ -49,9 +49,9 @@ where
         Self {
             amount: value.amount.into(),
             spec: PositionSpecDTO::new_internal(
-                value.liability,
-                value.min_asset.into(),
-                value.min_sell_asset.into(),
+                value.spec.liability(),
+                value.spec.min_asset().into(),
+                value.spec.min_trasaction_amount().into(),
```

Comment:
what's wrong here:
1. we're trying to minimize the usage of accessors ... whereas here we introduce a couple
2. we notice that `value.spec` is used multiple times ... then what shall we do?